winui/Rating/Placeholder-value.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Placeholder value in WinUI Rating control | Syncfusion
+description: Learn all about the placeholder feature in the Rating control to show the average rating value before the user provide their own rating.
```

Comment:
Description: Learn about the placeholder feature in the Rating control to show the average rating value before the user provides their rating.

Reply:
Updated the above mentioned changes.